src/main/java/uk/gov/hmcts/reform/civil/utils/NocNotificationUtils.java

```diff
@@ -0,0 +1,80 @@
+package uk.gov.hmcts.reform.civil.utils;
+
+import uk.gov.hmcts.reform.civil.enums.CaseRole;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+
+public class NocNotificationUtils {
+
+    private static String otherSolicitor1Email;
+    private static String otherSolicitor1OrgId;
+    private static String otherSolicitor2Email;
+    private static String otherSolicitor2OrgId;
+
+    private NocNotificationUtils() {
+        //NO-OP
+    }
+
+    public static String getPreviousSolicitorEmail(CaseData caseData) {
+        return caseData.getChangeOfRepresentation().getFormerRepresentationEmailAddress();
+    }
+
+    private static void setOtherSolicitor1(CaseData caseData) {
+        if (isRespondent2NewSolicitor(caseData)) {
```

Comment:
why is it that only when Respondent2 Solicitor is changed, we notify applicantsolicitor?

Shouldn't all be applicantsolicitor?